repository_service_tuf_worker/repository.py

```diff
@@ -106,11 +106,31 @@ def refresh_settings(self, worker_settings: Optional[Dynaconf] = None):
             settings = self._worker_settings
         else:
             settings = worker_settings
-
+        #
         # SQL
-        settings.SQL = rstuf_db(self._worker_settings.SQL_SERVER)
+        #
+        sql_server_url = self._worker_settings.get("SQL_SERVER")
+        # clean 'postgresql://' if present
+        sql_server = sql_server_url.replace("postgresql://", "")
+        if sql_user := self._worker_settings.get("SQL_USER"):
+            if self._worker_settings.SQL_PASSWORD.startswith("/run/secrets"):
```

Comment:
If we miss `SQL_PASSWORD` here do we want to log the error better or just leave `AttributeError` as our error of choice?
Maybe if we raise an error we can say: `SQL_PASSWORD required because SQL_USER was used`.
But maybe the error `Settings' object has no attribute 'SQL_PASSWORD` is good enough.

Reply:
Filed issue:
- https://github.com/repository-service-tuf/repository-service-tuf-worker/issues/284